### PR TITLE
fixed #102

### LINKED ISSUE #102: Starboard should resend the image

### PR DESCRIPTION
**Pull request by sourTaste000**
this branch is based on #100 lol so if you merge this one dont forget to close #100

### CODE REVIEW

**Comment by sourTaste000** [comment]
ill close #100

**Comment by sourTaste000** [comment]
oops typo in commiit

**Comment by sourTaste000** [comment]
:thonkkill:

**Comment by dominikaaaa** [comment]
Ty, i'll take a look later today

File: commands/issue.js [hunk 1]
```diff
@@ -20,8 +20,10 @@ module.exports.run = async (client, message, args) => {
                     .setThumbnail(result.user.avatar_url)
                     .setDescription(result.body)
                     .addField("Assignee", `${result.assignee.login}`)
+                    //someone fix these
```
Comment (sourTaste000) [inline commands/issue.js:23]: i need to fix this

**Comment by humboldt123** [comment]
Sourtaste showed me that it works on the test bot.

**Pull request merged**